emails/template-utils.js

```diff
@@ -36,6 +36,11 @@ function asset(path) {
   return `${assetBasePath}/${path}`
 }
 
+function stripWhitespace(str) {
```

Comment:
Should this be something like `trimStartMultiline`, to express that it’s at the start and multiline? Also trying to imitate https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimStart

Reply:
Updated in 5186287.

Made it so we trimmed all starting and ending spaces off of new lines :).